src/shared/components/case-editor/services/cases.service.ts

```diff
@@ -343,4 +345,18 @@ export class CasesService {
           });
     }
   }
+
+  /*
+  Checks if the user has role of pui-case-manager and returns true or false
+  */
+  private isPuiCaseManager(): boolean {
+    const userInfoStr = this.sessionStorageService.getItem('userDetails');
+    if (userInfoStr) {
+      const userInfo: UserInfo = JSON.parse(userInfoStr);
+      if (userInfo && userInfo.roles.indexOf('pui-case-manager') !== -1) {
```

Comment:
please add a check to see if userInfo.roles is present.

Reply:
Also, move 'pui-case-manager' to a public constant

Reply:
ok, will do